rbi/annotations/graphql.rbi

```diff
@@ -1,12 +1,5 @@
 # typed: true
 
-module GraphQL
```

Comment:
Removing the older `lhm` versions on my machine solved this.

Reply:
Yeah I restored it incorrectly. Fixed now, with `filename` added.

But if this is already defined in the GraphQL RBI, then why do we also need it in the annotation?

Reply:
One of the use cases of `rbi-central` is to add hand written signatures to methods since tapioca doesn't generate sigs, [example](https://github.com/Shopify/tapioca/blob/3341a9b328b0b2535a7f2790489adfec643dbbc3/sorbet/rbi/gems/graphql%402.2.7.rbi#L59).

Reply:
Ah, got it.